hyprctl/main.cpp

```diff
@@ -22,37 +22,54 @@
 #include <filesystem>
 #include <stdarg.h>
 
-const std::string USAGE = R"#(usage: hyprctl [(opt)flags] [command] [(opt)args]
-
-commands:
-    monitors
-    workspaces
-    activeworkspace
-    clients
-    activewindow
-    layers
-    devices
-    binds
-    dispatch
-    keyword
-    version
-    kill
-    splash
-    hyprpaper
-    reload
-    setcursor
-    getoption
-    cursorpos
-    switchxkblayout
-    seterror
-    setprop
-    plugin
-    notify
-    globalshortcuts
-    instances
-
-flags:
+const std::string USAGE = R"#(usage:  hyprctl [flags] [<command> [args]]
+        hyprctl --batch {<command 1> [args] ; <command 2> [args] ; ...}
+
+LISTING COMMANDS:
+    monitors:           List outputs
+    workspaces:         List all workspaces
+    activeworkspace:    Get currently active workspace
+    clients:            List clients (e.g. windows)
+    activewindow:       Get currently active window
+    layers:             List layers
+    animations:         List animations and bezier curves in use
+    devices:            List devices
+    binds:              List registered binds
+    instances:          List running Hyprland instances
+    globalshortcuts:    List global shortcuts
+    version:            Print hyprland version
+
+CONFIGURATION COMMANDS:
+    keyword <keyword> [args]:   Execute a keyword
+    getoption <option>:         Get value of <option>
+    reload:                     Reload configurations
+
+PLUGIN:
+    plugin list:            List loaded plugins
+    plugin load <path>:     Load plugin from <path>
+    plugin unload <path>:   Unload plugin at <path>
+
+THEMING:
+    hyprpaper <keywords>        Issue hyprpaper keywords using IPC
```

Comment:
hyprpaper should not be in theming, maybe a separate category called "integrations"?